docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md

```diff
@@ -1,90 +1,86 @@
+- A free account on nuget.org. Follow the instructions at [Add a new individual account](../nuget-org/individual-accounts.md#add-a-new-individual-account).
 
 ## Create a class library project
 
-You can use an existing .NET Class Library project for the code you want to package, or create a simple one as follows:
+You can use an existing .NET Class Library project for the code you want to package, or create a simple project as follows:
 
-1. Create a folder called `AppLogger`.
+1. Create a folder named *AppLogger*.
+1. Open a command prompt and switch to the *AppLogger* folder. All the dotnet CLI commands in this quickstart run on the current folder by default.
+1. Enter `dotnet new classlib`, which creates a project with the current folder name.
 
-1. Open a command prompt and switch to the `AppLogger` folder.
-
-1. Type `dotnet new classlib`, which uses the name of the current folder for the project.
-
-   This creates the new project.
+For more information, see [dotnet new](/dotnet/core/tools/dotnet-new).
 
 ## Add package metadata to the project file
 
-Every NuGet package needs a manifest that describes the package's contents and dependencies. In a final package, the manifest is a `.nuspec` file that is generated from the NuGet metadata properties that you include in the project file.
+Every NuGet package has a manifest that describes the package's contents and dependencies. In the final package, the manifest is a *.nuspec* file, which uses the NuGet metadata properties you include in the project file.
 
-1. Open your project file (`.csproj`, `.fsproj` or `.vbproj` depending on the language you're using) and add the following minimal properties inside the existing `<PropertyGroup>` tag, changing the values as appropriate:
+Open the *.csproj*, *.fproj*, or *.vbproj* project file, and add the following properties inside the existing `<PropertyGroup>` tag. Use your own values for name and company, and replace the package identifier with a unique value.
 
-    ```xml
-    <PackageId>AppLogger</PackageId>
-    <Version>1.0.0</Version>
-    <Authors>your_name</Authors>
-    <Company>your_company</Company>
-    ```
+```xml
+<PackageId>Contoso.08.28.22.001.Test</PackageId>
+<Version>1.0.0</Version>
+<Authors>your_name</Authors>
+<Company>your_company</Company>
+```
 
-    > [!Important]
-    > Give the package an identifier that's unique across nuget.org or whatever host you're using. For this walkthrough we recommend including "Sample" or "Test" in the name as the later publishing step does make the package publicly visible (though it's unlikely anyone will actually use it).
+> [!Important]
+> The package identifier must be unique across nuget.org or other host. Publishing makes the package publicly visible, so if you use the example AppLogger library or other test library, use a unique name that includes `Sample` or `Test`.
```

Comment:
```suggestion
> The package identifier must be unique across nuget.org and other package sources. Publishing makes the package publicly visible, so if you use the example AppLogger library or other test library, use a unique name that includes `Sample` or `Test`.
```

The "or" is true only if package consumers don't use multiple package sources. If a package consumer uses multiple package sources, the package ID must be unique across all of them.